lib/init/features/firestore.js

```diff
@@ -27,11 +34,56 @@ var _initRules = function(setup, config) {
       type: "input",
       name: "rules",
       message: "What file should be used for Firestore Rules?",
-      default: "firestore.rules",
+      default: DEFAULT_RULES_FILE,
     },
-  ]).then(function() {
-    return config.writeProjectFile(setup.config.firestore.rules, RULES_TEMPLATE);
-  });
+  ])
+    .then(function() {
+      var filename = setup.config.firestore.rules;
+
+      if (fsutils.fileExistsSync(filename)) {
+        var msg =
+          "File " +
+          chalk.bold(filename) +
+          " already exists." +
+          " Do you want to overwrite it with the Firestore Rules from the Firebase Console?";
+        return prompt.once({
+          type: "confirm",
+          message: msg,
+          default: false,
+        });
+      }
+
+      return Promise.resolve(true);
+    })
+    .then(function(overwrite) {
+      if (!overwrite) {
+        return Promise.resolve();
+      }
+
+      var projectId = setup.instance || setup.rcfile.projects.default;
```

Comment:
I feel like there's got to be a better way to get the project Id here?

Reply:
setup.instance is actually the database instance, so it's definitely not what you want.  setup.rcfile.projects.default would work, but is a little clunky.

The best thing to do is to add a "projectId" field to the "setup" object during project selection, you'd want to do it in 2 places:
- https://github.com/firebase/firebase-tools/blob/master/lib/init/features/project.js#L35
- https://github.com/firebase/firebase-tools/blob/master/lib/init/features/project.js#L35